home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,31 @@
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_move.jpg)                              | Switches to a mode that makes it possible to [move](../machines/selecting-and-moving-objects.md) the selected object(s) by using the mouse; if the mode is active, the icon will appear blue |
-| ![](<../../../.gitbook/assets/iVP\_icon\_3D\_rotate (1).jpg>)                      | Switches to a mode that makes it possible to [rotate](../machines/scale-and-rotate-objects.md#rotate-objects) the selected object(s); if the mode is active, the icon will appear blue       |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_scale.jpg)                             | Switches to a mode that makes it possible to [scale](../machines/scale-and-rotate-objects.md#scale-objects) the selected object(s); if the mode is active, the icon will appear blue         |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_screenshot.jpg)                        | Takes a [screenshot](../advanced-tools/creating-screenshots.md) of the 3D-view                                                                                                                                        |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_FOV\_settings.jpg)                     | Changes the field of view of the currently chosen camera                                                                                                                                     |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_reset\_camera\_view.jpg)               | Resets the camera view to a centered default perspective                                                                                                                                     |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_add\_current\_camera\_view\_point.jpg) | Adds a new [virtual camera](../advanced-tools/virtual-cameras.md)                                                                                                                            |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_update\_selected\_camera.jpg)          |                                                                                                                                                                                              |
+| ---------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+ ![](../../../.gitbook/assets/iVP\_icon\_3D\_toggle_camera_mode.png)                                | Toggles the camera mode, which simulates a traversal between the positions of your placed cameras
```

Comment:
This still does not explain what the different camera modes actually do. The transition does not need to be mentioned here.